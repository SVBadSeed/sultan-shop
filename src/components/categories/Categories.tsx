import React from 'react'

const FilterCart = () => {


    return (
        <ul className='catalog-top__filter'>
            <li> Уход <br/> за телом</li>
            <li> Уход <br/> за руками</li>
            <li> Уход <br/> за ногами</li>
            <li> Уход <br/> за лицом</li>
            <li> Уход <br/> за волосами</li>
            <li> Средства <br/>для загара</li>
            <li> Средства <br/> для бритья</li>
            <li> Подарочные <br/> наборы</li>
            <li> Гигиеническая <br/> продукция</li>
            <li> Гигиена <br/> полости рта</li>
            <li> Бумажная <br/>продукция</li>
        </ul>
    )
}

export default FilterCart