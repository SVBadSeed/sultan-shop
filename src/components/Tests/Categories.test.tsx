import {render} from "@testing-library/react"
import Categories from "../categories/Categories"
import * as reduxHooks from 'react-redux'

jest.mock('react-redux')

const mockedUseSelector = jest.spyOn(reduxHooks, 'useSelector')
describe('Categories', () => {
    it('should create Categories with id', () => {
        mockedUseSelector.mockReturnValue('')

        const component = render(<Categories/>)

        expect(component).toMatchSnapshot()
    })
    it('should create Categories with categories', () => {
        mockedUseSelector.mockReturnValue('0')

        const component = render(<Categories/>)

        expect(component).toMatchSnapshot()
    })
})