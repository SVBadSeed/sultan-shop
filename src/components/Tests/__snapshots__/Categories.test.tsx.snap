// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Categories should create Categories with categories 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <ul
        class="categories"
      >
        <li
          class=""
        >
          Уход за телом
        </li>
        <li
          class=""
        >
          Уход за руками
        </li>
        <li
          class=""
        >
          Уход за ногами
        </li>
        <li
          class=""
        >
          Уход за лицом
        </li>
        <li
          class=""
        >
          Уход за волосами
        </li>
        <li
          class=""
        >
          Средства для загара
        </li>
        <li
          class=""
        >
          Подарочные наборы
        </li>
        <li
          class=""
        >
          Гигиеническая продукция
        </li>
        <li
          class=""
        >
          Гигиена полости рта
        </li>
        <li
          class=""
        >
          Бумажная продукция
        </li>
      </ul>
    </div>
  </body>,
  "container": <div>
    <ul
      class="categories"
    >
      <li
        class=""
      >
        Уход за телом
      </li>
      <li
        class=""
      >
        Уход за руками
      </li>
      <li
        class=""
      >
        Уход за ногами
      </li>
      <li
        class=""
      >
        Уход за лицом
      </li>
      <li
        class=""
      >
        Уход за волосами
      </li>
      <li
        class=""
      >
        Средства для загара
      </li>
      <li
        class=""
      >
        Подарочные наборы
      </li>
      <li
        class=""
      >
        Гигиеническая продукция
      </li>
      <li
        class=""
      >
        Гигиена полости рта
      </li>
      <li
        class=""
      >
        Бумажная продукция
      </li>
    </ul>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Categories should create Categories with id 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <ul
        class="categories"
      >
        <li
          class=""
        >
          Уход за телом
        </li>
        <li
          class=""
        >
          Уход за руками
        </li>
        <li
          class=""
        >
          Уход за ногами
        </li>
        <li
          class=""
        >
          Уход за лицом
        </li>
        <li
          class=""
        >
          Уход за волосами
        </li>
        <li
          class=""
        >
          Средства для загара
        </li>
        <li
          class=""
        >
          Подарочные наборы
        </li>
        <li
          class=""
        >
          Гигиеническая продукция
        </li>
        <li
          class=""
        >
          Гигиена полости рта
        </li>
        <li
          class=""
        >
          Бумажная продукция
        </li>
      </ul>
    </div>
  </body>,
  "container": <div>
    <ul
      class="categories"
    >
      <li
        class=""
      >
        Уход за телом
      </li>
      <li
        class=""
      >
        Уход за руками
      </li>
      <li
        class=""
      >
        Уход за ногами
      </li>
      <li
        class=""
      >
        Уход за лицом
      </li>
      <li
        class=""
      >
        Уход за волосами
      </li>
      <li
        class=""
      >
        Средства для загара
      </li>
      <li
        class=""
      >
        Подарочные наборы
      </li>
      <li
        class=""
      >
        Гигиеническая продукция
      </li>
      <li
        class=""
      >
        Гигиена полости рта
      </li>
      <li
        class=""
      >
        Бумажная продукция
      </li>
    </ul>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
